draft-tuexen-tsvwg-dtls-chunk-key-management.xml

```diff
@@ -110,15 +110,62 @@ they appear in all capitals, as shown here.</t>
 
 <section>
 <name>Architecture</name>
-<t>TBD</t>
+<t>This document describes how Datagram Transport Layer Security (DTLS) 1.3 is
+used to establish keys for securing SCTP using the DTLS Chunk. This approach
+combines the performance and encryption flexibility of DTLS Chunks with the
+integrated key management capabilities of DTLS. The resulting solution
+can be viewed from two perspectives: as a new key management method for
+SCTP, and as an SCTP-specific encryption layer integrated with DTLS.
```

Comment:
My point was that the SCTP encryption is not done with DTLS. This was the idea from Ericsson. The point is that we run
* DTLS on top of SCTP without modifications including a DTLS record protector.
* A DTLS chunk protector. But this is not integrated in DTLS. It provides a similar function, but it uses different keys.